omnibus/omnibus-test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ueo pipefail
+
+channel="${CHANNEL:-unstable}"
+product="${PRODUCT:-chef-server}"
```

Comment:
Because this script lives vendored into the Supermarket repo, should the `product` variable maybe default to `supermarket`? Or just _be_ `supermarket`?

Or keep this as close to a common script as possible so updates to it can be dropped in with minimal differences?

Reply:
great catch! it should be changed to supermarket. i'll fix that